panorama/matcher.py

```diff
@@ -7,8 +7,8 @@ class matcher:
     def __init__(self):
         #self.surf = cv2.SURF_create()
         self.sift = cv2.SIFT_create()
-        FLANN_INDEX_KDTREE = 0
-        index_params = dict(algorithm=0, trees=5)
+        FLANN_INDEX_KDTREE = 1
+        index_params = dict(algorithm=FLANN_INDEX_KDTREE, trees=5)
```

Comment:
what is FLANN_INDEX_KDTREE for?

Reply:
It just used as algorithm paramaters.
Because the class doesn't use `FLANN_INDEX_KDTREE` variable, so I just put it back.
But I am not sure why your algorithm use `0`.
Most examples use `1`.